packages/teams-js/src/public/dialog.ts

```diff
@@ -21,7 +21,15 @@ import {
 import { runtime } from './runtime';
 
 /**
- * Namespace to interact with the dialog module-specific part of the SDK.
+ * This group of capabilities enables apps to show dialogs. There are two primary types of dialogs: URL-based dialogs and [Adaptive Card](https://learn.microsoft.com/adaptive-cards/) dialogs.
+ * Both types of dialogs are shown on top of your app, preventing interaction with your app while they are displayed.
+ * - URL-based dialogs allow you to specify a URL for which the contents will be shown inside the dialog.
+ *   - For URL dialogs, you should use the functions and interfaces in the {@link dialog.url} namespace.
+ * - Adaptive Card-based dialogs allow you to provide JSON describing an Adaptive Card that will be shown inside the dialog.
+ *   - For Adaptive Card dialogs, you should use the functions and interfaces in the {@link dialog.adaptiveCard} namespace.
+ *
+ * @remarks Note that dialogs were previously called "task modules". While they have been renamed for clarity, the functionality has been maintained.
+ * For more details about dialogs, see [Task Modules](https://learn.microsoft.com/microsoftteams/platform/task-modules-and-cards/what-are-task-modules)
```

Comment:
We're [in the process](https://github.com/MicrosoftDocs/msteams-docs/pull/7853) of updating Teams docs from Task Module-> Dialog terminology. How about:

For more details, see [Dialogs](https://learn.microsoft.com/microsoftteams/platform/task-modules-and-cards/what-are-task-modules).